src/Navbar.tsx

```diff
@@ -1,13 +1,99 @@
+import { NavLink } from "react-router-dom";
+import logo from "../public/img/logo.png";
+import { useState, useEffect, useRef, LegacyRef } from "react";
+import { LuMenu } from "react-icons/lu";
+import { CgClose } from "react-icons/cg";
+
 const Navbar = () => {
-    return (
-    <div className = "fixed"> 
-        <ol>
-            <li><a href="/">home</a></li>
-            <li><a href="/menu">menu</a></li>
-            <li><a href="/about">about</a></li>
-        </ol>
-    </div>
-    )
-  }
-
-export default Navbar
\ No newline at end of file
+  const [isSideBarOpen, setIsSideBarOpen] = useState<boolean>(false);
+  let sidebarRef: LegacyRef<HTMLDivElement> = useRef(null);
```

Comment:
This should use `const` instead of `let`. You should also use `RefObject` instead of `LegacyRef`.